medicaltorch/filters.py

```diff
@@ -9,13 +9,13 @@ def __init__(self, filter_empty_mask=True,
 
     def __call__(self, sample):
         input_data, gt_data = sample['input'], sample['gt']
-        
+
         if self.filter_empty_mask:
             if not np.any(gt_data):
                 return False
-        
+
         if self.filter_empty_input:
-            if not np.any(input_data):
+            if not np.all(np.any(input_data, axis=0)):
```

Comment:
Great :D!

Reply:
In fact, i'm having a problem with this line: input_data is a list, so if the first column of one image is all zero -> it returns False even if the image is not all zeros. Do you agree with this instead: `if not np.all([np.any(img) for img in input_data])`

Reply:
Yep! Alternatively `np.all(np.any(np.array(input_data), axis=0))` could work too.